src/lib/icons/audio/MuteIcon.svelte

```diff
@@ -1,7 +1,12 @@
 <script lang="ts">
+    // Mute from https://fonts.google.com/icons
+    // Filled = 1
+    // Weight = 400
+    // Grade = 0
+    // Optical size = 24px
     export let _class = 'fill-black-100';
 </script>
 
-<svg class={_class} xmlns="http://www.w3.org/2000/svg" height="20" width="20"
+<svg class={_class} xmlns="http://www.w3.org/2000/svg" height="24" width="24"
     ><path d="M5.833 12.5V7.5H9.042L13.333 3.333V16.667L9.042 12.5Z" /></svg
```

Comment:
This one has the same data but just has the height and width changed.  Please regenerate with optical size = 24px